federation-js/src/composition/validate/postComposition/__tests__/providesNotOnEntity.test.ts

```diff
@@ -131,7 +131,7 @@ describe('providesNotOnEntity', () => {
           "code": "PROVIDES_NOT_ON_ENTITY",
           "locations": Array [
             Object {
-              "column": 3,
+              "column": 32,
```

Comment:
In this case, the existence of `@provides` on this field is incorrect, so I think we should point to the directive and not its `fields` argument.